aeneas/src/vst/Parser.v3

```diff
@@ -132,6 +132,14 @@ component Parser {
 					return true;
 				}
 			}
+			'l' => {
```

Comment:
I had a look back at our whiteboard conversation, and I think we should use the 'struct' keyword for fixed layouts. Can you update this PR to use 'struct' for the keyword and name the new Vst node `VstStruct`?

Reply:
Got it. All keyword 'layout' changed to 'struct', methods renamed.